### ISSUE TITLE
Use @page

### ISSUE THREAD

**Issue by akrost** [Afur/flutter_html_to_pdf#27]
Hi,

is there a way to use @page to set the page margin and add a page counter? Something like this:

```
@page {
    size: A4;
    margin: 25mm 25mm 25mm 25mm;  
    @bottom-right {
        content: counter(page) " of " counter(pages);
    }
}
```

If it's not currently supported it would be a great improvement, I think.

**Comment by rlbartz** [comment]
I'm waitig for this feature too. Maybe add support for Paged.JS framework too.

**Comment by Afur** [comment]
Yeah, the problem is that we're using default printer api provided by the system itself. I'm not sure if we can make with this.
